deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -630,6 +630,16 @@ private void ingestWork(ContentContainerObject parent, Resource parentResc, Reso
 
                 // Cease refreshing the transaction
                 txRefresher.stop();
+            } catch (ChecksumMismatchException e) {
+                txRefresher.interrupt();
+                tx.cancelAndIgnore();
+
+                if (retries > 0) {
+                    retries--;
+                    ingestWork(parent, parentResc, childResc, retries);
```

Comment:
Could you add a `warn` level message stating that there was a checksum mismatch, probably include the message from the error? So that we see if this issue is occurring